libmamba/tests/src/doctest-printer/flat_set.hpp

```diff
@@ -1,23 +0,0 @@
-// Copyright (c) 2023, QuantStack and Mamba Contributors
-//
-// Distributed under the terms of the BSD 3-Clause License.
-//
-// The full license is in the file LICENSE, distributed with this software.
-
-#include <doctest/doctest.h>
-#include <fmt/format.h>
-#include <fmt/ranges.h>
-
-#include "mamba/util/flat_set.hpp"
-
-namespace doctest
-{
-    template <typename K, typename C, typename A>
-    struct StringMaker<mamba::util::flat_set<K, C, A>>
```

Comment:
We should delete this one, because Catch2 defines StringMaker for `is_range` types (also defined in Catch2)